.yarn/patches/@trezor-connect-web-npm-9.3.0-040ab10d9a.patch

```diff
@@ -31,3 +42,16 @@ index 9b13c370a5ac8b4e4fc0315ed40cdf615d0bb0cb..4dbd97fc28df49beb73379451974ec48
                      useCore: true,
                  },
              });
+@@ -292,9 +294,11 @@ class PopupManager extends events_1.default {
+                 this.popupPromise = undefined;
+             }
+             (_b = this.iframeHandshakePromise) === null || _b === void 0 ? void 0 : _b.promise.then(payload => {
++                var modifiedSettings = Object.assign({}, this.settings);
++                modifiedSettings.env = 'webextension';
+                 this.channel.postMessage({
+                     type: events_2.POPUP.INIT,
```

Comment:
This second call to `events_2.POPUP.INIT` was not present in v9.2.2, but it is present here in v9.3.0

See https://npmfs.com/package/@trezor/connect-web/9.2.2/lib/popup/index.js